src/invidious/videos.cr

```diff
@@ -455,7 +455,7 @@ struct Video
 
                       qualities.each do |quality|
                         json.object do
-                          json.field "url", rv["author_thumbnail"]?.try &.gsub(/s\d+-/, "s#{quality}-")
+                          json.field "url", rv["author_thumbnail"].try &.gsub(/s\d+-/, "s#{quality}-")
```

Comment:
Since you removed the `?` you don't need the `.try` anymore as it will never be nil

Reply:
Ah, yep, good catch!

Reply:
```suggestion
                          json.field "url", rv["author_thumbnail"].gsub(/s\d+-/, "s#{quality}-")
```